test/test_implicit_free_surface_solver.jl

```diff
@@ -55,11 +55,11 @@ end
 @testset "Implicit free surface solver tests" begin
     for arch in archs
 
-        rectilinear_grid = RegularRectilinearGrid(size = (128, 1, 1),
+        rectilinear_grid = RegularRectilinearGrid(size = (128, 1, 5),
                                                   x = (0, 1000kilometers), y = (0, 1), z = (-400, 0),
                                                   topology = (Bounded, Periodic, Bounded))
 
-        lat_lon_grid = RegularLatitudeLongitudeGrid(size = (90, 90, 1),
+        lat_lon_grid = RegularLatitudeLongitudeGrid(size = (90, 90, 5),
```

Comment:
Is `2` too small?

Reply:
Its kind of nice to have some points in the middle maybe? The run time should only go with the ReducedField size I think?

Reply:
If we wanted to be fancy we could do two tests - a one layer and an N layer, both with the same depth. The solution, iterations counts and runtime etc... should be the same for both. Would that be too fancy?

Reply:
Touche about runtime.

That does sound like a nice test to add (in a future PR) that a 2D and 3D problem with the same 2D structure have the same behavior!

Reply:
👍